src/connectors/index.ts

```diff
@@ -18,10 +17,11 @@ interface NetworkConnectorArguments {
 }
 
 const POLLING_INTERVAL = 10000
+const DEFAULT_CHAIN_ID = 1
 
 const networkConnectorArguments: NetworkConnectorArguments = {
   urls: [],
-  defaultChainId: CHAIN_ID,
+  defaultChainId: DEFAULT_CHAIN_ID,
```

Comment:
Why is the default needed?

If it is really needed, I would prefer to check which one of the following envs is set:
```
if (NETWORK_URL_MAINNET)
if (NETWORK_URL_RINKEBY) 
if (NETWORK_URL_XDAI) 
```
and then use the corresponding network Id, instead of hardcoding a solution

Reply:
This should be fixed now. Maybe show some kind of error/message when any URL is defined.